examples/id.js

```diff
@@ -1,22 +1,39 @@
 /* eslint no-console: 0 */
 'use strict'
 
-var ipfsd = require('../')
+const factory = require('../')
```

Comment:
Same as above.

Reply:
these examples don't really have a package.json - I'll move them into their own directory and add it.